lib/mocha.js

```diff
@@ -437,7 +437,7 @@ Mocha.prototype.loadFiles = function (fn) {
  *   .then(() => mocha.run(failures => process.exitCode = failures ? 1 : 0))
  *   .catch(() => process.exitCode = 1);
  */
-Mocha.prototype.loadFilesAsync = function () {
+Mocha.prototype.loadFilesAsync = function ({esmDecorator} = {}) {
```

Comment:
Not sure how this works. The default `esmDecorator` is `undefined` and thus calling it will throw an exception.

Reply:
`esmDecorator` just passed through to the function `formattedImport` in `esm-utils.js`. And there it has default value

Reply:
Right! I missed that. Me and others... so in the interest of the code documenting itself, can you move the default `forward` function in `esm-utils` to be what defines the default for `esmDecorator` in `Mocha.prototype.loadFilesAsync`.

In other words, move the default up in the call chain.

Reply:
Unfortunately `esmUtils.requireOrImport` is used elsewhere in the code, so default value for `esmDecorator` should be set at the level of `esmUtils`.
I think in fact it pretty obvious: no decorator passed - no any decoration